src/encoded/static/components/cart/cart.js

```diff
@@ -1113,6 +1114,8 @@ DatasetFacets.propTypes = {
     selectedDatasetCount: PropTypes.number,
     /** Callback when the user clicks on a file format facet item */
     termClickHandler: PropTypes.func.isRequired,
+    /** Facet-loading progress for progress bar, or null if not displayed */
+    facetLoadProgress: PropTypes.number,
```

Comment:
Should you use PropTypes.oneOfType instead of PropTypes.number to cover the null case?